lib/auth/permissions.go

```diff
@@ -278,6 +279,7 @@ func GetCheckerForBuiltinRole(clusterName string, clusterConfig services.Cluster
 							services.NewRule(services.KindRole, services.RO()),
 							services.NewRule(services.KindClusterAuthPreference, services.RO()),
 							services.NewRule(services.KindClusterConfig, services.RO()),
+							services.NewRule(services.KindProxyConfig, services.RW()),
```

Comment:
Why does this proxy need RW to it's config? Can it be changed live?

Reply:
ah, great catch, that's not needed actually, was used before, not any more

Reply:
fixed, take a look